index.html

```diff
@@ -0,0 +1,184 @@
+          <h3 class="title-main">Introduction</h3>
+          <p class="program-pa">The orchestra master will give a presentation about the works to be performed and the invited soloists</p>
+        </li>
+        <li class="program-items">
+          <img class="concert" src="./images/284-2848248_related-wallpapers-transparent-white-music-notes-png.png" alt="concert">
+          <h3 class="title-main concert-ti">Concert</h3>
+          <p class="program-pa concert-pa">The Vienna Mozart Orchestra was founded in 1986 by musicians from the best Viennese orchestras and chamber music ensembles. <p>
+        </li>
+        <li class="program-items">
+          <img class="intermission" src="./images/interm.png" alt="">
+          <h3 class="title-main">Intermission</h3>
+          <p class="program-pa">A chance for performers and audience to take a break. The musicians leave the stage; you may leave your seat.</p>
+        </li>
+        <li class="program-items">
+          <img class="concert" src="./images/284-2848248_related-wallpapers-transparent-white-music-notes-png.png" alt="concert">
+          <h3 class="title-main concert-ti">Concert</h3>
+          <p class="program-pa">The Vienna Mozart Orchestra was founded in 1986 by musicians from the best Viennese orchestras and chamber music ensembles. <p>
+        </li>
+        <li class="program-items">
+          <img class="galery" src="./images/galery.png" alt="">
+          <h3 class="title-main gallery-ti">Gallery</h3>
+          <p class="program-pa gallery-pa">Opening of the gallery in honor of Guarneri and Stradivarius, see the most expensive violin in the world.</p>
+        </li>
+      </ul>
+    </article>
+  </section>
+  
+  <section class="soloists">
+    <div class="main-header" id="soloists-header">
+      <h2 class="main-title">Soloists</h2>
+      <div class="main-line"></div>
+    </div>
+    <div class="soloists-cards">
+
+    </div>
+  </section>
+
+  <section class="partners">
+    <article class="sponsors">
+      <div class="main-header">
+        <h2 class="main-title title-sponsor">Partners</h2>
+        <div class="partners-line"></div>
+      </div>
+      <ul class="sponsores-img">
+        <li class="images">
+          <img class="image-sponsore" src="./images/rehoboth.png" alt="rehoboth">
+        </li>
+        <li class="images">
+          <img class="image-sponsore" src="./images/mozilla.png" alt="mozilla">
+        </li>
+        <li class="images">
+          <img class="image-sponsore" src="./images/line-friends.png" alt="logo2">
+        </li>
+        <li class="images">
+          <img class="image-sponsore" src="./images/lumen.png" alt="lumen">
+        </li>
+      </ul>
+    </article>
+  </section>
+
+  <footer class="footer">
+    <img class="footer-img" src="./images/music3.jpg" alt="logo2">
+    <p>All rights reserved <br> this web page is made only whit academic purposes</p>
+  </footer>
```

Comment:
- [ ] Kindly remove the sponsors section from the About page to match the original design.
- [ ] Please change the footer background color on the About page to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/163407826-e421274d-4d24-443c-b142-ef7160f6be9e.png) | ![image](https://user-images.githubusercontent.com/62290545/163408207-5ec804a3-1bdd-471b-8649-874a742d67a6.png) |

Reply:
thanks for your suhgeestions 
i just have made the required changes 

![image](https://user-images.githubusercontent.com/95243996/163421716-7c383dc5-a6ba-438b-9a6a-145b504a79bb.png)